internal/channel/fanout.go

```diff
@@ -0,0 +1 @@
+package channel
```

Comment:
Is this file suppose to be empty?

Reply:
I considered breaking fanout out of the agent, but at the moment it's still stuck in there. Part of the issue is that the fanout is not channel based. so maybe this should just go away for now.

Reply:
Ah ok, if it isn't essential for the rest of the config api work just removing it for now sounds like a good idea.